src/wrappers/azure.js

```diff
@@ -0,0 +1,14 @@
+import BaseWrapper from './base-wrapper'
+import qq from 'fine-uploader/lib/core/azure'
+import { traditional as callbackNames } from './callback-names'
```

Comment:
Oh! One thing missing here we need a set of "azure" callback names. I'll fix and merge.